wordpress-mobile/gutenberg-mobile
# Add block insertion flow e2e test

See Gutenberg PR for more information: https://github.com/WordPress/gutenberg/pull/27381

To test:

- CI: Test iOS/Android on Device - Canaries tests should only run files with `@canary` in their names
- CI: Full E2E Test suite should pass
- Locally: `npm run test:e2e:ios:local` and `npm run test:e2e:android:local` should work as before.

PR submission checklist:
